assets/javascripts/popper.js

```diff
@@ -191,7 +192,7 @@ function getOffsetParent(element) {
   var noOffsetParent = isIE(10) ? document.body : null;
 
   // NOTE: 1 DOM access here
-  var offsetParent = element.offsetParent;
+  var offsetParent = element.offsetParent || null;
```

Comment:
May I ask what is this for? :thinking: just trying to find out what the new version changes and I am curious why this is needed. :wink:

Reply:
The changelog is here: https://github.com/FezVrasta/popper.js/releases

All the questions about the actual code should be asked there as well.
This repo simply packages the upstream code as a rubygem.